util/http.h

```diff
@@ -6,26 +6,31 @@
 #include <string>
 
 namespace atlas {
-
 namespace util {
 
 class http {
  public:
+  http(int connect_timeout_seconds, int read_timeout_seconds)
```

Comment:
You no longer have to update the get/post calls for a new option, but this still requires you to update all places where the client is created whenever a new option is added.

Reply:
Fair point. I was thinking more about adding new methods to the client, but you're right. Fixed in the latest commit.